glusterd2/commands/volumes/volume-create.go

```diff
@@ -42,6 +43,10 @@ func validateVolCreateReq(req *api.VolCreateReq) error {
 		return gderrors.ErrInvalidVolName
 	}
 
+	if gutils.IsReservedKeyword(req.Name) {
+		return fmt.Errorf("invalid name, volume name cannot be among glusterd reserved keywords:%v", gutils.ReservedKeywords)
```

Comment:
`glusterd` or `glusterd2`?

Reply:
That's a point I want to discuss, will users be always referring the glustercli daemon as gluster daemon 2 or is it just temperory.

Reply:
@aravindavk ^^

Reply:
you can skip that word itself. "invalid name, volume name cannot be among reserved keywords"